lib/plugins/create/create.test.js

```diff
@@ -251,28 +229,6 @@ describe('Create', () => {
         expect(dirContent).to.include('.gitignore');
       });
     });
-    it('should generate scaffolding for "aws-alexa-typescript" ' +
```

Comment:
This test is unnecessary since we test the same at the top of the file in a more generic way.